arangod/Agency/AgencyFeature.cpp

```diff
@@ -245,7 +245,7 @@ void AgencyFeature::start() {
   // Find the agency prefix:
   auto feature = ApplicationServer::getFeature<ClusterFeature>("Cluster");
   if (!feature->agencyPrefix().empty()) {
-    arangodb::consensus::Supervision::setAgencyPrefix(
+    consensus::Supervision::setAgencyPrefix(
```

Comment:
I never approve just consmetic changes :). There is really no necessity. It also only mangles git blame.